common/src/main/java/org/wso2/testgrid/common/util/StringUtil.java

```diff
@@ -47,4 +49,15 @@ public static String concatStrings(Object... objects) {
         }
         return stringBuilder.toString();
     }
+
+    public static String generateRandomString(int characterCount) {
```

Comment:
Shall we add a java doc comment here describing the purpose of this method for future reference?

Reply:
Added..